readthedocs/organizations/managers.py

```diff
@@ -31,21 +31,15 @@ def teams_for_user(self, user, organization, admin, member):
 
         return teams.distinct()
 
-    def admin(self, user, organization=None, include_all=False):
-        if user.is_superuser and include_all:
-            return self.get_queryset().all()
-
+    def admin(self, user, organization=None):
         return self.teams_for_user(
             user,
             organization,
             admin=True,
             member=False,
         )
 
-    def member(self, user, organization=None, include_all=False):
-        if user.is_superuser and include_all:
```

Comment:
In all other querysets we return all for superusers, but I'm removing it since that code was never executed (`include_all=True`).